pyproject.toml

```diff
@@ -18,13 +18,17 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools]
 packages = ["torchqdynamics"]
 
-[tool.black]
-line-length = 80
-preview = true
-target-version = ["py38"]
-skip-string-normalization = true
+[tool.yapf]
+based_on_style = "facebook"
+column_limit = 88
+indent_width = 4
+SPLIT_BEFORE_NAMED_ASSIGNS = false
+DEDENT_CLOSING_BRACKETS = true
+SPLIT_ARGUMENTS_WHEN_COMMA_TERMINATED = false
+INDENT_DICTIONARY_VALUE = true
 
 [tool.isort]
 profile = "black"
-line_length = 80  # to be consistent with black
+line_length = 88  # to be consistent with yapf
 skip = ["__init__.py"]
+
```

Comment:
Could we remove the extra line?

Reply:
You can tune your IDE so it does all this little things for you, for example in VSCode user configuration
```json
"files.trimTrailingWhitespace": true,
"files.insertFinalNewline": true,
"files.trimFinalNewlines": true,
"files.eol": "\n",
```

Reply:
Added that in my VSCode settings, thanks !